src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -381,3 +381,88 @@ def linear_isometry_equiv.from_orthogonal_span_singleton
   (n : ℕ) [fact (finrank 𝕜 E = n + 1)] {v : E} (hv : v ≠ 0) :
   (𝕜 ∙ v)ᗮ ≃ₗᵢ[𝕜] (euclidean_space 𝕜 (fin n)) :=
 linear_isometry_equiv.of_inner_product_space (finrank_orthogonal_span_singleton hv)
+
+section complex
```

Comment:
no longer need this section name!

Reply:
I renamed; sectioning seems like good practice?  I know this keeps my personal variable declarations local.